src/System.Data.Common/ref/Configurations.props

```diff
@@ -5,6 +5,7 @@
       netcoreapp;
       netfx;
       uap;
+      net471;
```

Comment:
Are these new configurations necessary? I thought you were planning to just reship a subset of the binaries we already had. Is there any reason that wouldn't just work?

Reply:
as we discussed offline, they are required for most cases since we need to make sure we type forward whenever possible. I just added a change that will make sure we type forward whenever a type is now inbox.